docs/browsers-api/index.md

```diff
@@ -2,7 +2,48 @@
 sidebar_label: API
 ---
 
-# API Reference
+# @puppeteer/browsers
+
+Manage and launch browsers/drivers from a CLI or programmatically.
+
+## CLI
+
+Use `npx` to run the CLI without installing:
+
+```sh
+npx @puppeteer/browsers --help
+```
+
+or install the package as a dependency and run it from your `package.json` script:
+
+```sh
+npm i @puppeteer/browsers
+```
+
+```json
+{
+  "scripts": {
+    "browsers": "@puppeteer/browsers --help"
+  }
+}
+```
+
+CLI help will provide all documentation you need to use the CLI.
+
+```sh
+npx @puppeteer/browsers --help # help for all commands
+npx @puppeteer/browsers install --help # help for the install command
+npx @puppeteer/browsers launch --help # help for the launch command
+```
+
+Known limitations:
+
+1. We support installing and running Firefox and Chrome/Chromium. The `latest` keyword only works during the installation. For the `launch` command you need to specify an exact build ID. The build ID is provided by the `install` command.
```

Comment:
Provider an example for the install output.

Reply:
Added a reference to the help message (I don't want to duplicate the format in multiple places).